proto/kvrpcpb.proto

```diff
@@ -863,6 +863,8 @@ message TimeDetail {
     // cannot be excluded for now, like Mutex wait time, which is included in this field, so that
     // this field is called wall time instead of CPU time.
     int64 process_wall_time_ms = 2;
+    // KV read wall Time means the time used in key/value scan and get.
+    int64 kv_read_wall_time_ms = 3;
```

Comment:
What's the difference between `process_wall_time_ms` and `kv_read_wall_time_ms`?

Is `kv_read_wall_time_ms` with same semantics as `process_wall_time_ms`, but for special kind of read requests, i.e. `get` and `batch_get`?

Reply:
In coprocessor they are different.